drive/drive_test.go

```diff
@@ -0,0 +1,459 @@
+package drive
```

Comment:
Try moving tests to `drive_test` package to get a good sense of what the exported API feels like when working with it. That would suffice wrinkles like the options struct I pointed out earlier. Same for other tests.